content/master/guides/mrd-activation-policies.md

```diff
@@ -0,0 +1,689 @@
+  # Caching
+  - clusters.elasticache.aws.crossplane.io
+  - instances.memorystore.gcp.crossplane.io
+  - caches.redis.azure.crossplane.io
+```
+
+### Networking services
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: networking-services
+spec:
+  activations:
+  # Core networking
+  - vpcs.ec2.aws.crossplane.io
+  - subnets.ec2.aws.crossplane.io  
+  - networks.compute.gcp.crossplane.io
+  - subnetworks.compute.gcp.crossplane.io
+  - virtualnetworks.network.azure.crossplane.io
+  
+  # Load balancing
+  - "*.elbv2.aws.crossplane.io"
+  - "*.compute.gcp.crossplane.io"
+  - loadbalancers.network.azure.crossplane.io
+  
+  # DNS and routing
+  - "*.route53.aws.crossplane.io"
+  - "*.dns.gcp.crossplane.io" 
+  - zones.dns.azure.crossplane.io
+```
+
+## Team-based policies
+
+### Platform team
+
+Broad access for platform engineering:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: platform-team-resources
+spec:
+  activations:
+  # Infrastructure management
+  - "*.iam.aws.crossplane.io"
+  - "*.iam.gcp.crossplane.io"
+  - "*.authorization.azure.crossplane.io"
+  
+  # Networking and security
+  - "*.ec2.aws.crossplane.io"
+  - "*.compute.gcp.crossplane.io"
+  - "*.network.azure.crossplane.io"
+  
+  # Monitoring and logging
+  - "*.cloudwatch.aws.crossplane.io"
+  - "*.monitoring.gcp.crossplane.io"
+  - "*.insights.azure.crossplane.io"
+```
+
+### App team
+
+Resources needed by app developers:
+
+```yaml
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: app-team-resources
+spec:
+  activations:
+  # Compute resources
+  - instances.ec2.aws.crossplane.io
+  - clusters.eks.aws.crossplane.io
+  - clusters.gke.gcp.crossplane.io
+  
+  # Storage
+  - buckets.s3.aws.crossplane.io
+  - buckets.storage.gcp.crossplane.io
+  - accounts.storage.azure.crossplane.io
+  
+  # Databases (read-only access through compositions)
+  - "*.rds.aws.crossplane.io"
+  - instances.sql.gcp.crossplane.io
+```
+
+## Dynamic activation patterns
+
+### Conditional activation
+
+Use multiple policies to create conditional resource activation:
+
+```yaml
+# Base resources always active
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: base-resources
+spec:
+  activations:
+  - instances.ec2.aws.crossplane.io
+  - buckets.s3.aws.crossplane.io
+
+---
+# Optional resources for advanced features
+apiVersion: apiextensions.crossplane.io/v1alpha1
+kind: ManagedResourceActivationPolicy
+metadata:
+  name: advanced-resources
+spec:
+  activations:
+  - clusters.eks.aws.crossplane.io
+  - "*.lambda.aws.crossplane.io"
+```
+
+You can delete the `advanced-resources` policy to deactivate optional 
+resources while keeping base capability.
+
+### Feature flag patterns
+
+Use labels and naming to create feature flag-like behavior:
```

Comment:
How? Don't they just all roll-up to a singleton MRAP policy regardless of labels?